requirements.txt

```diff
@@ -11,6 +11,7 @@ unidecode
 pucas>=0.6
 piffle>=0.3
 djiffy>=0.5
+pytest-runner
```

Comment:
this should go in dev-requirements

Reply:
this installation was required for `annotator_store` 🥴